src/components/custom/_signup-public.js

```diff
@@ -339,8 +339,8 @@ const SignupPublic = ({
     return (
         <SectionContainer>
             <Show.Desktop>
-                <Wrapper>
-                    <Container>
+                <Container>
+                    <Wrapper>
```

Comment:
It seems like the orange section should have a border radius of 8px. Could you add that in as well please? 🙏